components/ResultTable/matchResultTable.data.js

```diff
@@ -0,0 +1,96 @@
+export const testMatch = {
```

Comment:
Let's use mocks instead.

Reply:
For sure. I just didn't have that at the time.